kernel/string/convert_test.c

```diff
@@ -403,3 +370,25 @@ TEST (KernelString, ConvertFromBoolean) {
 	ASSERT_STR(expect, result);
 	free(result);
 }
+
+TEST (String, ConvertToFloat) {
```

Comment:
Please create variable to store expected and actual

Reply:
I remember you give me an advice that I should use 1 ASSERT for 1 test case is best.

Reply:
yes, 1 ASSERT type for 1 test case is best practice
but i mean
`double expected = `0.000000000000006;`
`double actual = string_to_double((char*);`
`ASSERT_DBL_NEAR(expected, actual);`

Reply:
Ok, I will change it for this class.